app/views/articles/index.html.erb

```diff
@@ -1,10 +1,10 @@
-<h1>Articles</h1>
+<h1><%= t :articles %></h1>
```

Comment:
Please, use `()` for method call and send arguments for translation in the same style everywhere (I mean as strings)